src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.cpp

```diff
@@ -1514,6 +1514,56 @@ std::set<std::vector<element::Type>> jit_sigmoid_emitter::get_supported_precisio
     [[maybe_unused]] const std::shared_ptr<ov::Node>& node) {
     return {{element::f32}};
 }
+/// Greater///
+jit_greater_emitter::jit_greater_emitter(jit_generator* host, cpu_isa_t host_isa, const element::Type exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {}
+
+jit_greater_emitter::jit_greater_emitter(jit_generator* host, cpu_isa_t host_isa, const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, get_arithmetic_binary_exec_precision(node)) {}
+
+size_t jit_greater_emitter::get_inputs_num() const {
+    return 2;
+}
+
+size_t jit_greater_emitter::aux_vecs_count() const {
+    return 1;
+}
+
+size_t jit_greater_emitter::aux_fp_gprs_count() const {
+    return 0;
+}
```

Comment:
Since this class has `aux_fp_gprs_count()` with the same implementation as the base class [`jit_emitter`](https://github.com/openvinotoolkit/openvino/blob/85ea61f17abfc48751ea62ecde4c81ea043d8eb8/src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_emitter.cpp#L63), no need to override this method then. Please remove this overrided method